wger/settings.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
I'm not sure we need to create the settings file this way. This file should be automatically created with the `invoke create-settings` and the path can be specified with the option `--settings-path`. This shouldn't block merging though

Reply:
@collin5 , the settings file was recreated in the right way. Thanks